Lib/statistics.py

```diff
@@ -93,8 +93,8 @@
 0.75
 >>> correlation(x, y)  #doctest: +ELLIPSIS
 0.31622776601...
->>> linear_regression(x, y)  #doctest: +ELLIPSIS
-(1.5, 0.099999999999...)
+>>> linear_regression(x, y)  #doctest:
```

Comment:
Maybe use the following to avoid decimal representation issues? Or use different values for the example so that "+ELLIPSIS" is enough.

```python3
>>> '%.2f, %.2f' % linear_regression(x, y)
'1.50, 0.10'
```

Reply:
It seems to work now.

Reply:
Yes, but tests that sometimes fail can require a lot of effort down the road. We should make every effort to make tests robust.